### PR TITLE
[PLAT-4818] Add manual testing mode to Maze-runner test fixture

### PR DESCRIPTION
**Pull request by Cawllec**
## Goal

Enables easier development and testing of the notifier and associated E2E tests by enabling them to be run locally.

## Tests
Run several scenarios locally.

### CODE REVIEW

**Comment by Cawllec** [comment]
There appears to be an error in building due to the removal of a `null` from the maze-runner scenarios project. This will be address ASAP before merging.

**Review by twometresteve — COMMENTED** [review]
I haven't had a chance to exercise this dynamically, but on visual inspection it certainly looks like the functionality that I was expecting to see.  I'd be happy to see this merged once the build passes and approved by another.

File: tests/features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/MainActivity.kt [hunk 1]
```diff
@@ -14,6 +15,8 @@ import com.bugsnag.android.mazerunner.scenarios.Scenario
 
 class MainActivity : Activity() {
 
+    private val apiKeyKey = "BUGSNAG_API_KEY"
```
Comment (fractalwrench) [inline tests/features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/MainActivity.kt:18]:
```suggestion
    private val apiKey = "BUGSNAG_API_KEY"
```

**Review by fractalwrench — APPROVED** [review]

File: tests/features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/MainActivity.kt (hunk 1 above)
Comment (twometresteve) [reply tests/features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/MainActivity.kt:18]: Although it looks like a typo I think this is actually correct - "BUGSNAG_API_KEY" is the key for getting the apiKey.

**Review by twometresteve — APPROVED** [review]
LGTM.  I built and tested the fixture locally and it behaved as I expected, with errors appearing in the dashboard and apiKey saved/cleared accordingly.

File: tests/features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/MainActivity.kt (hunk 1 above)
Comment (fractalwrench) [reply tests/features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/MainActivity.kt:18]: Ah - I'd assumed that this was a variable that holds the API key. Perhaps something like the following would make it more clear that it's a constant:

`private const val EXTRA_API_KEY = "BUGSNAG_API_KEY"`

**Comment by twometresteve** [comment]
Although I've approved this, should it actually be going into `next`?

**Pull request merged**